src/providers/hooks.ts

```diff
@@ -1,4 +1,6 @@
 import { useContext } from 'react';
-import { UserContext } from './contexts';
+import { OrganizationContext, UserContext } from './contexts';
 
 export const useUser = () => useContext(UserContext);
+
+export const useOrganization = () => useContext(OrganizationContext);
```

Comment:
One way to remove the `!!` is to maybe introduce 2 hooks.
```
export const useOrganization = () : ServerOrganization => useContext(OrganizationContext) as ServerOrganization;
export const useRawOrganization = () => useContext(OrganizationContext);
```
The second one can be used in TopBarContent since it is used in the orphaned user path.
Could be hacky but perhaps ok?

Reply:
On rethinking.. since the major use-case is fetching the selectedOrganizationOnly.. 
ignore the above comment.
Add a new hook:
```
export const useOrganization = () : ServerOrganization => useContext(OrganizationContext).selectedOrganization as ServerOrganization;
```
Maybe rename the original `useOrganization` to something else for the smaller use-cases that need all the data.
In all usecases:
```
  const organization = useOrganization();
```
This would be a sweeping change.. but just some feedback.

Reply:
The last option would be to initialize the default org as non-null with some dummy values:
{
  name: '',
  id: -1,
 ...
}

and if id is -1, assume no org (only place this matters is TopBarContent)

Reply:
I think I understand what you're getting at. However, might the first one need to look like this, perhaps?
```typescript
export const useSelectedOrg = () => useContext(OrganizationContext).selectedOrganization as ServerOrganization;
```

Reply:
oop, didn't see your later message

Reply:
okay, that's making sense. Thanks so much for your suggestions! I'll get something in there